tests/app/views/test_questionnaire.py

```diff
@@ -8,7 +8,7 @@
 from app.data_model.questionnaire_store import QuestionnaireStore
 from app.schema_loader.schema_loader import load_schema_file
 from app.questionnaire.location import Location
-from app.views.questionnaire import update_questionnaire_store_with_answer_data, update_questionnaire_store_with_form_data
+from app.views.questionnaire import update_questionnaire_store_with_answer_data, update_questionnaire_store_with_form_data, _remove_empty_household_members_from_answer_store
```

Comment:
If this really is to be considered a private method, I don't think we should be testing it outside of the view using it.